### ISSUE TITLE
[Blood Elf][NPC] Felendren the Banished

### ISSUE THREAD

**Issue by dredd-sunwell** [SunwellWoW/Sunwell-TBC-Bugtracker#26]
Decription:  NPC ID 15367 has missing dialogue, upon aggroing he should say "Take heart! Your friends will not long mourn your passing!" with voiceover /script PlaySoundFile("Sound\\Creature\\FelendrenTheBanished\\A_FelendrenTakeHeart.wav")

He should also cast Mind Flay ID 16568

**Comment by Kokaina** [comment]
Fixed and implemented.
